test/images/porter/Makefile

```diff
@@ -5,6 +5,9 @@
 # `make container` will build a container-- you must supply a tag.
 # `make push` will push the container-- you must supply a tag.
 
+# Set this to the *next* version to prevent accidentally overwriting the existing image.
```

Comment:
Oops, definitely didn't want to do that. I do think using the current git commit would be more helpful than a md5 of the executable, if only for figuring out when in time a particular tag referred to. Updated `make tag` to use that instead... let me know what you think